reri_err_reporting.adoc

```diff
@@ -349,11 +358,24 @@ the signal configured in the `ces` field. When `cece` is 1, the logging of a CE
 in the error record does not cause an error signal and an error signal
 configured in `ces` occurs only on a `cec` overflow.
 
+The set-read-in-progress (`srdp`) field, when written with a value of 1, causes
+the `rdip` (read-in-progress) bit of the associated `status_i` register to be
```

Comment:
So `srdp` is not ever cleared, why is it always read as 0, why not have it return its current value?

Reply:
`srdp` is a control bit. `rdip` is the status bit. The `srdp` always reads 0.

Reply:
I should have been more clear. There are two types of bits in the control register - control bits and configuration bits. Whereas hardware implements storage for configuration bits - e.g., UES are backed by flops, the control bits are command bits - they trigger a certain action in hardware but otherwise there is no storage associated with those bits. The `srdp` has an action of setting `rdip` when written to 1. Both `srdp` and `sinv` are such control bits which do not have storage associated with them and on read return 0.